node-process-argv/echo.js

```diff
@@ -0,0 +1,2 @@
+console.log('Hello');
+console.log('Adrienne');
```

Comment:
For this one we want to log `process.argv[2]` so that the program can print back anything that was typed, like in the example.